### ISSUE TITLE
The plug-in shows an error on a specific subroutine

### ISSUE THREAD

**Issue by gcartier94** [BroadcomMFD/rexx-language-support#3]
I have a specific subroutine in some of my Rexx codes called Parse_Input_Parameters, it will basically parse an input string and create a dynamic stem out of it:

```rexx
Parse_Input_Parameters: Procedure Expose input_parm.
  input_string = Arg(1)
  number_of_parms = Length(Space(input_string, 0)) - Length(Space(Translate(input_string, ' ', ';'), 0))
  Do number_of_parms
    Parse Var input_string key '=' value ';'
    Interpret "input_parm."Strip(key) "="  value
    parm_end_pos = Pos(';', input_string)
    input_string = Substr(input_string, parm_end_pos + 1)
  End
  Return
```

But I'm getting an error on the following lines:
![image](https://user-images.githubusercontent.com/34777337/88812506-2c7a5f80-d18e-11ea-8832-8bae8bee9c93.png)
![image](https://user-images.githubusercontent.com/34777337/88812400-0ce33700-d18e-11ea-8895-93c05a1aae8b.png)

The routine itself works just fine. 

Any thoughts on this?

Thanks!

**Comment by ishche** [comment]
Thank you for your feedback!
My guess is that currently, our grammar doesn't understand `value` as an identifier but as a keyword. It should be fixed.

**Comment by ishche** [comment]
We just released 0.0.14 with fix for this issue.
Can you please confirm that it works for you?

**Comment by gcartier94** [comment]
Hello, the problem seems to be fixed!

Thank you for the quick response! 😎
